lib/source/google_key_source.ex

```diff
@@ -26,4 +26,29 @@ defmodule ExFirebaseAuth.KeySource.Google do
     end)
     |> Enum.filter(fn {_, value} -> not is_nil(value) end)
   end
+
+  defp get_max_age(headers) do
+    headers |> get_cache_control_header() |> get_get_max_age_from_cache_control()
+  end
+
+  defp get_cache_control_header(headers) do
+    Enum.find(headers, fn {k, _v} -> k == "cache-control" end)
+  end
+
+  defp get_get_max_age_from_cache_control(nil) do
```

Comment:
It looks like you made a small typo here

```suggestion
  defp get_max_age_from_cache_control(nil) do
```